widgetastic_manageiq/__init__.py

```diff
@@ -5744,7 +5744,9 @@ def fill(self, value):
 
         # Some EntryPoint need to apply tree selection
         if self.tree.is_displayed:
-            self.apply.click()
+            if include_domain:
+                self.parent.modal.include_domain.click()
```

Comment:
hummm.. `self.parent` means you are using `View` which holding `EntryPoint` right?
I think we need to include that checkbox as per of this widget.